tests/functional/monitoring/prometheus/alerts/test_alert_mds_cache_high_usage.py

```diff
@@ -0,0 +1,211 @@
+        log.info(
+            "Metadata IO started in the background. Lets wait for 15 minutes before validating the MDS alert"
+        )
+        time.sleep(900)
+        log.info("Validating the alert now")
+        assert active_mds_alert_values(threading_lock)
+
+        node_name = cluster.get_active_mds_info()["node_name"]
+
+        # Unschedule active mds running node.
+        unschedule_nodes([node_name])
+        log.info(f"node {node_name} unscheduled successfully")
+
+        # Drain node operation
+        drain_nodes([node_name])
+        log.info(f"node {node_name} drained successfully")
+
+        # Make the node schedule-able
+        schedule_nodes([node_name])
+        log.info(f"Scheduled the node {node_name}")
+        log.info("Script will sleep for 10 minutes before validating the alert")
+        time.sleep(600)
+        assert active_mds_alert_values(threading_lock)
+
+    @pytest.mark.polarion_id("OCS-5577")
+    def test_mds_cache_alert_with_active_node_scaledown(
+        self, run_metadata_io_with_cephfs, threading_lock
+    ):
+        # This function verifies the mds cache alert with active mds scale down and up
+        time.sleep(900)
+        assert active_mds_alert_values(threading_lock)
+
+        active_mds = cluster.get_active_mds_info()["mds_daemon"]
+        deployment_name = "rook-ceph-mds-" + active_mds
+        log.info(f"Scale down {deployment_name} to 0")
+        helpers.modify_deployment_replica_count(
+            deployment_name=deployment_name, replica_count=0
+        )
+        time.sleep(100)
+        log.info(f"Scale up {deployment_name} to 1")
+        helpers.modify_deployment_replica_count(
+            deployment_name=deployment_name, replica_count=1
+        )
+
+        time.sleep(900)
+        assert active_mds_alert_values(threading_lock)
+
+    @pytest.mark.polarion_id("OCS-5578")
+    def test_mds_cache_alert_with_sr_node_scaledown(
+        self, run_metadata_io_with_cephfs, threading_lock
+    ):
+        # This function verifies the mds cache alert with standby-replay mds scale down and up
+        time.sleep(900)
+        assert active_mds_alert_values(threading_lock)
+
+        sr_mds = cluster.get_mds_standby_replay_info()["mds_daemon"]
+        deployment_name = "rook-ceph-mds-" + sr_mds
+        helpers.modify_deployment_replica_count(
+            deployment_name=deployment_name, replica_count=0
+        )
+        time.sleep(100)
+        helpers.modify_deployment_replica_count(
+            deployment_name=deployment_name, replica_count=1
+        )
+
+        time.sleep(900)
+        assert active_mds_alert_values(threading_lock)
+
+    @pytest.mark.polarion_id("OCS-5579")
+    def test_mds_cache_alert_with_all_mds_node_scaledown(
+        self, run_metadata_io_with_cephfs, threading_lock
+    ):
+        # This function verifies the mds cache alert with both active and standby-replay mds scale down and up
+        time.sleep(900)
```

Comment:
Please add logger.info message before every time.sleep that we are doing, so we can identify timeouts in the log console, including the sleep period. Same as you did at [tests/functional/monitoring/prometheus/alerts/test_alert_mds_cpu_high_usage.py](https://github.com/red-hat-storage/ocs-ci/pull/9757/files#diff-092955c44cfe901fa9242fe90704a92a42b7dcdabb650727da505351ac4a2cea)

 ```
log.info(
            "File creation IO started in the background."
            " Script will sleep for 15 minutes before validating the MDS alert"
        )

```

Reply:
Implemented the changes.